src/app/examples/modal-example/first-embedded-modal-example/first-embedded-modal-example.component.ts

```diff
@@ -0,0 +1,34 @@
+import { EventData } from 'tns-core-modules/data/observable';
+import { Component, ViewContainerRef } from '@angular/core';
+
+import { ModalService } from '~/kirby/components/modal/services/modal-service';
+import { EmbeddedModalComponent } from '~/kirby/components/modal/embedded-modal.component';
+import { ModalConfig } from '~/kirby/components/modal/config/modal-config';
+import { SecondEmbeddedModalExampleComponent } from '../second-embedded-modal-example/second-embedded-modal-example.component';
+
+@Component({
+  templateUrl: './first-embedded-modal-example.component.html',
+  providers: [ModalService],
+})
+export class FirstEmbeddedModalExampleComponent implements EmbeddedModalComponent {
+  uid: number;
```

Comment:
This never get's assigned... (I know it does somewhere else, but it's not very clear)

Reply:
completely agree with you, this is now done better